docs/static/breaking-changes.asciidoc

```diff
@@ -3,54 +3,82 @@
 
 This section discusses the changes that you need to be aware of when migrating your application to Logstash {version}.
 
-**Application Settings:** Introduced a new way to configure application settings for Logstash through a settings.yml file. This file
+[float]
+=== Changes in Logstash Core
+
+* **Application Settings:** Introduced a new way to configure application settings for Logstash through a settings.yml file. This file
 is typically located in `LS_HOME/config`, or `/etc/logstash` when installed via packages. Logstash will not be able
 to start without this file, so please make sure to pass in `--path.settings` if you are starting Logstash manually
 after installing it via a package (RPM, DEB).
 
-**Release Packages:** When Logstash is installed via DEB, RPM packages, it uses `/usr/share/logstash` and `/var/lib/logstash` to install binaries and config files
+* **Release Packages:** When Logstash is installed via DEB, RPM packages, it uses `/usr/share/logstash` and `/var/lib/logstash` to install binaries and config files
```

Comment:
When we say, `config files` here, won't that be a bit confusing, if we already said that the configuration for RPM/DEB goes in `/etc/logstash`? 

I'm thinking that this might be better worded as a "plugin metadata" path.